File: Day 07/Homework/homework4.py
number = int(input("Enter any number: "))

if number % 9 == 0 and number > 100:
    print("the number is the multiple of nine and also the number is > then 100")
elif number % 9 == 0 and number < 100:
    print("the number is multiple of nine, but it is < then 100")
elif number % 9 != 0 and number > 100:
    print("the number is not the multiple of nine, but it is > then 100")
elif number % 9 != 0 and number < 100:
    print("the number is not the multiple of nine and also, it is < then 100")

File: Day 090/Homework/homework4.js
// 4. მომხმარებელს შემოატანინეთ რიცხვი და შეამოწმეთ ლუწია თუ კენტი.

let usernum = Number(prompt("Enter any number: "))

if(usernum % 2 === 0) {
    console.log(usernum, "is even")
}
else{
    console.log(usernum, "is odd")
}